src/main/java/org/kohsuke/github/GitHubBuilder.java

```diff
@@ -89,7 +89,11 @@ public static GitHubBuilder fromEnvironment(String loginVariableName, String pas
     	
     	if (oauthValue != null)
     		env.put("oauth", oauthValue);
-    	
+
+        Object endPoint = System.getenv(endpointVariableName);
```

Comment:
Sorry about the wrong indentation, will fix it. All `System.getenv` was taken in as Object, that's why. Do I change all to String?